src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -287,6 +287,18 @@ public function viewAction(int $orderId, Request $request): Response
             'order_id' => $orderId,
         ]);
 
+        $backOfficeOrderButtons = new ActionsBarButtonsCollection();
```

Comment:
Must be a service or something extendable

Reply:
I dont think 🤔 this is a collection of buttons, intended to be propagated through a hook to all modules which wants to add its button to be displayed. It's like DoctrineCollection, you would not make it a service, would you ?

Reply:
It's like an array but I did want to type-hint it

Reply:
I also think the service definition would be overkill, however there should be an interface for the collection and button class (even though we provide concrete classes)

Reply:
Agreed with @jolelievre